app/views/custom/layouts/_admin_header.html.erb

```diff
@@ -0,0 +1,42 @@
+<header class="header">
+  <div class="top-links">
+    <div class="expanded row">
+      <%= render "shared/locale_switcher" %>
+
+      <%= link_to root_path, class: "float-right" do %>
+        <%= t("admin.dashboard.index.back", org: setting["org_name"]) %>
+      <% end %>
+    </div>
+  </div>
+
+  <div class="expanded row admin-top-bar">
+    <div class="title-bar" data-responsive-toggle="responsive_menu" data-hide-for="medium">
+      <button class="menu-icon" type="button" data-toggle="responsive_menu"></button>
+      <div class="title-bar-title"><%= t("application.menu") %></div>
+    </div>
+
+    <div class="top-bar">
+      <div class="top-bar-title">
+        <% if namespace == "officing" %>
+          <%= link_to "#", accesskey: "0" do %>
+            <%= image_tag(image_path_for("logo_header.png"), class: "float-left", alt: t("layouts.header.logo")) %>
+          <% end %>
+        <% else %>
+          <%= link_to namespaced_root_path, accesskey: "0" do %>
```

Comment:
Is there a reason why we're removing the `<h1>` tag present in the original file?

Reply:
Because before we showed a text and now an image, that was my motive.
Do you think it's better to keep the h1?

Reply:
Ah `<h1>` tag is OK IMHO. Actually CONSUL now uses `<h1>` in the application layout too: consul@bb2e2fa0a.